src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -0,0 +1,331 @@
+            cols="12"
+            sm="3"
+          >
+            <!-- empty column to line up with Business Information above -->
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <MessageBox color="red">
+              <strong>Important:</strong> This extraprovincial registration is historical.
+              Please contact BC Registries staff.
+            </MessageBox>
+
+            <RegistriesContactInfo class="mt-6" />
+          </v-col>
+        </v-row>
+      </template>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { isEqual } from 'lodash'
+import { getName } from 'country-list'
+import { BusinessLookupResultIF, EmptyBusinessLookup } from '@/interfaces'
+import ExtraproBusinessLookup from './ExtraproBusinessLookup.vue'
+import MessageBox from '@/components/common/MessageBox.vue'
+import RegistriesContactInfo from '@/components/common/RegistriesContactInfo.vue'
+
+@Component({
+  components: {
+    ExtraproBusinessLookup,
+    MessageBox,
+    RegistriesContactInfo
+  }
+})
+export default class ExtraproRegistration extends Vue {
+  @Getter(useStore) getContinuationInBusinessInfo!: any
+  @Getter(useStore) getShowErrors!: boolean
+
+  @Action(useStore) setContinuationInBusinessInfo!: (x: any) => void
+
+  // Local properties
+  active = false
+  business = null // FUTURE: update BusinessLookupResultIF with new properties
+  checkbox = false
+
+  readonly checkboxLabel = 'I understand and acknowledge that the extraprovincial registration of this' +
+    ' business will be cancelled and made historical in the B.C. Registry once the continuation in' +
+    ' application has been submitted.'
+
+  /** Whether we have a looked-up business. */
+  get haveLookupBusiness (): boolean {
+    return !isEqual(this.business, EmptyBusinessLookup)
+  }
+
+  /** The identifier. */
+  get identifier (): string {
+    return this.business.identifier
+  }
+
+  /** The jurisdiction. */
+  get jurisdiction (): string {
+    const hj = this.business.homeJurisdiction
+    if (hj?.country) {
+      const country = getName(hj.country)
+      const region = (hj.region === 'FEDERAL' ? 'Federal' : hj.region)
+      if (region) return `${region}, ${country}`
+      return country
+    }
+    return null
+  }
+
+  /** The business name. */
+  get businessName (): string {
+    return this.business.name
+  }
+
+  /** The business number. */
+  get businessNumber (): string {
+    return this.business.bn
+  }
+
+  /** The incorporation date. */
+  get incorporationDate (): string {
+    // FUTURE: format date if needed
+    return this.business.registrationDate
+  }
+
+  /** Whether business is Active (otherwise it's Historical). */
+  get isBusinessActive (): boolean {
+    return (this.business.status === 'ACTIVE')
```

Comment:
This is the only status we care about (as I assume the only other status is HISTORICAL), so probably no need to create an enum.

Later, if we get a definitive list of possible statuses, we can create an enum.